Content/RootScript/BaseClasses/Utility/Util.as

```diff
@@ -0,0 +1,152 @@
+// Script: RootScript/BaseClasses/World.as
+// Purpose: Generic baseclasses such as World::Model with a model and coordinates.
+
+
+namespace Util {
+	shared int tick;
+	shared class Model {
+		::Model@ mesh;
```

Comment:
Use more descriptive identifiers to avoid root namespace resolution

Reply:
Name conflict is avoided with ::[global]. I wasn't sure what else to call it, as any and all references to Model, unless within Util, must be preceded by Namespace::class. Open to discussion!

Reply:
On further thought, because "Model" is such a generic term, perhaps it would be better to rename the internal model class to CModel, denoting it is the C-level model class?

Reply:
I have renamed the C-level namespace/class to CModel.